lib/pdk/validators/base_validator.rb

```diff
@@ -25,17 +25,29 @@ def self.parse_targets(options)
                     options[:targets]
                   end
 
-        targets.map { |target|
+        skipped = []
+        invalid = []
+        matched = targets.map { |target|
           if respond_to?(:pattern)
             if File.directory?(target)
-              Array[pattern].flatten.map { |p| Dir.glob(File.join(target, p)) }
+              target_list = Array[pattern].flatten.map { |p| Dir.glob(File.join(target, p)) }
```

Comment:
At this point if the target directory contains a mixture of file types; we glob those that match the pattern and ignore those that don't.
We don't report those unmatching files as skipped... I think that's OK.